### PR TITLE
fix: added displayName and __ui__ to avatar components

### PR DESCRIPTION
**Pull request by esh2n**
Closes https://github.com/yamada-ui/yamada-ui/issues/2851 

## Description

Added displayName and __ui__ to avatar related components.

## Is this a breaking change (Yes/No):

No.

### CODE REVIEW

**Comment by esh2n** [comment]
Since some of the CI tests are failing, I will aim to fix them by today or tomorrow.

File: packages/components/avatar/src/avatar-icon.tsx
Comment (illionillion) [inline packages/components/avatar/src/avatar-icon.tsx:1]: import `FC` from `@yamada-ui/core`

Comment (esh2n) [reply packages/components/avatar/src/avatar-icon.tsx:1]: Thank you for teaching.
I fixed and request for re-reviewing.

File: packages/components/avatar/src/avatar.tsx
Comment (illionillion) [inline packages/components/avatar/src/avatar.tsx:1]: Please also fix the `AvatarImage`.

**Review by illionillion — COMMENTED** [review]
@esh2n 

The test is failing, so bring the fork repository up to date and merge it into your branch.

File: packages/components/avatar/src/avatar.tsx
Comment (esh2n) [reply packages/components/avatar/src/avatar.tsx:1]: Thank you for reviewing.

I added displayName for AvatarImage and merged main into branch.

**Review by hirotomoyamada — APPROVED** [review]

**Comment by hirotomoyamada** [comment]
Thanks!

**Pull request merged**